### PR TITLE
Include assets from "assets/" directory

### LINKED ISSUE #125: Include assets from `public` directory
Implement a catch all fallback of copying assets from a users `public` directory in the project (these will not be hashed).

### PR DESCRIPTION
**Pull request by mwistrand**
Resolves #125. Add a `copyPublicAssets` build flag used to determine whether files in
an optional `public/` directory should be copied to the output
directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ An array of paths to [CLDR JSON](https://github.com/dojo/i18n#loading-cldr-data)
 }
 ```
 
+#### `copyPublicAssets`: boolean
```
Comment (matt-gadd) [inline README.md]: Do we really need a flag for this given we can just presume if they have/haven’t a public folder they’re opting in/out?

**Comment by matt-gadd** [comment]
I think we are going to release this in the next major (under a group of enhancements), so I think this needs to be raised against the `next` branch?

File: README.md (hunk 1 above)
Comment (mwistrand) [reply README.md]: On second thought, I suppose we do not.

**Comment by matt-gadd** [comment]
@mwistrand can we put a feature in for the `test-app` for this?

**Comment by matt-gadd** [comment]
example for above: say an image tag with a src to the public directory?

**Pull request merged**